docs/en/reference/dql-doctrine-query-language.rst

```diff
@@ -1020,42 +1020,146 @@ Alternatively you can create an empty ``Query`` instance and invoke
     $q = $em->createQuery();
     $q->setDQL('select u from MyProject\Model\User u');
 
-Query Result Formats
-~~~~~~~~~~~~~~~~~~~~
+Query Result Formats (Hydration Modes)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The way in which the SQL result set of a DQL SELECT query is transformed
-to PHP is determined by the so-called "hydration mode":
-
--  **``getResult()``** (``HYDRATE_OBJECT``): Retrieves a collection of objects. The
-   result is either a plain collection of objects (pure) or an array
-   where the objects are nested in the result rows (mixed).
--  **``getSingleResult()``**: Retrieves a single object. If the
-   result contains more than one object, a ``NonUniqueResultException``
-   is thrown. If the result contains no objects, a ``NoResultException``
-   is thrown. The pure/mixed distinction does not apply.
--  **``getOneOrNullResult()``**: Retrieve a single object. If the
-   result contains more than one object, a ``NonUniqueResultException``
-   is thrown. If no object is found null will be returned.
--  **``getArrayResult()``** (``HYDRATE_ARRAY``): Retrieves an array graph (a nested
-   array) for read-only purposes.
-
-    .. note::
-
-        An array graph can differ from the corresponding object
-        graph in certain scenarios due to the difference of the identity
-        semantics between arrays and objects.
-
--  **``getScalarResult()``** (``HYDRATE_SCALAR``): Retrieves a flat/rectangular result
-   set of scalar values that can contain duplicate data. The
-   pure/mixed distinction does not apply.
--  **``getSingleScalarResult()``** (``HYDRATE_SINGLE_SCALAR``: Retrieves a single scalar
-   value from the result returned by the dbms. If the result contains
-   more than a single scalar value, a ``NonUniqueResultException`` is thrown. The
-   pure/mixed distinction does not apply.
-
-In parentheses are the constants of the ``Query`` class which you can use with the
-general-purpose method ``Query::execute(array $params = [], $hydrationMode = Query::HYDRATE_OBJECT)``.
-In fact, the methods in the list are just convenient shortcuts for the hydration mode.
+to PHP is determined by the so-called "hydration mode".
+
+getResult()
```

Comment:
Why are you not using backticks around this?

Reply:
I wasn't sure, cause it's a heading. Let's see how it looks now...